scripts/xe-enable-ipv6

```diff
@@ -23,20 +23,23 @@ fi
 . /etc/xensource-inventory
 
 if [ $1 = "enable" ]; then
-	rm /etc/modprobe.d/disable-ipv6 
-	
+	rm /etc/modprobe.d/disable-ipv6
```

Comment:
shouldn't the file be named `/etc/modprobe.d/disable-ipv6.conf`?
like here: https://github.com/xcp-ng/host-installer/blob/512877bfab35a729e4986658b8d71365235bbfb9/netutil.py#L438

Reply:
I would think so as well. the installer is definitely writing to `etc/modprobe.d/disable-ipv6.conf`.

Reply:
I can add the edit to my PR if you want. :)

Reply:
If you modify this, make sure to modify other references in the 'disable' flow:

```
echo "alias ipv6 no" > /etc/modprobe.d/disable-ipv6
echo "alias net-pf-10 off" >> /etc/modprobe.d/disable-ipv6
```

Reply:
On it!